state_epoch.go

```diff
@@ -0,0 +1,27 @@
+package verkle
+
+import (
+	"encoding/binary"
+)
+
+type StateEpoch uint64
+
+const (
+	NumActiveEpochs = 2
+)
+
+func EpochExpired(prev StateEpoch, cur StateEpoch) bool {
+	return cur-prev >= NumActiveEpochs
+}
+
+// Convert epoch to bytes
+func (e StateEpoch) Bytes() []byte {
+	b := make([]byte, 8)
+	binary.BigEndian.PutUint64(b, uint64(e))
+	return b
+}
```

Comment:
I would very much like not to have this method, it introduces an extra allocation+copy when this is used.